tests/test_web_log.py

```diff
@@ -43,13 +43,31 @@ def test_access_logger_format() -> None:
     Ref: https://stackoverflow.com/a/46102240/595220
     """,  # noqa: E501
 )
-def test_access_logger_atoms(mocker) -> None:
-    utcnow = datetime.datetime(1843, 1, 1, 0, 30)
-    mock_datetime = mocker.patch("aiohttp.datetime.datetime")
+def test_access_logger_time(monkeypatch) -> None:
```

Comment:
Instead of having a lot of copy-paste, just use `@pytest.mark.parametrize()`. Also, you now have the other test method without a proper skip marker.

Reply:
keep all pieces, only replace `extra` and `expected` dicts?

Reply:
yep